packages/quantic/docs/jsdoc-guidelines.md

```diff
@@ -0,0 +1,72 @@
+# JSDoc Guidelines for Quantic Components
+
+The Quantic library has [reference documentation](https://docs.coveo.com/en/quantic/latest/reference/) that is automatically generated from JSDoc comments in the component source code. The component comments are parsed by JSDoc and the resulting doclet data is parsed by our custom template script found in `quantic/docs/template/lwc-json/publish.js`. This document outlines the required attributes and format of these JSDOC comments.
+
+## Type Imports
+It is common to reference types from external dependencies, most notably Coveo Headless.
+Types should be defined via `typedef` before the class definition.
+
+example:
+```js
+/** @typedef {import("coveo").SearchEngine} SearchEngine */
+```
+
+## Components
+Components should be given a short but specific description explaining its role. Include relevant information about certain behaviours or limitations of the component here when pertinent to developers.
+
+### @fires
+Events fired from this component should be defined using the `@fires` tag.
+
+### @category
+Each component should be given one or more category values using the `@category` tag from one of the following cateogories:
+- **Search**: Components pertaining to the Search use-case.
+- **Case Assist**: Components pertaining to the Case Assist use-case.
+- **Result Template**: Components meant to be used within result templates.
+- **Utility**: Components providing a generic utility without pertaining to any specific Coveo Headless use-case.
+If there is more than one relevant category it should be specified with its own tag.
```

Comment:
I'm not sure what this means. Do you mean that components could have multiple tags? If that's the case, here's a suggestion:
```suggestion
If there are more than one relevant category, each should be specified with its own tag.
```